couchbase-ansible/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Advanced Linux : Deploy a Multi VM Couchbase Cluster with Ansible",
```

Comment:
Sure. I think Ansible won't get cut off with your title, but we can change later if it does.